ui/src/components/denali/InputDropdown.js

```diff
@@ -345,16 +346,19 @@ class InputDropdown extends React.Component {
                     initialSelectedItem={defaultSelectedItem}
                     inputValue={this.props.value}
                     itemToString={this.props.itemToString}
-                    onChange={(selected) => this.props.onChange(selected)}
+                    onSelect={(selected) => this.props.onChange(selected)} // onSelect seem to trigger more consistently than onChange
```

Comment:
using onSelect instead of onChange because it triggered more consistently during functional tests